editor/scene_tree_dock.cpp

```diff
@@ -2142,6 +2142,9 @@ void SceneTreeDock::_delete_confirm(bool p_cut) {
 
 	_push_item(nullptr);
 
+	// Fixes the EditorInspector still holding references to MultiNodeEdit objects.
+	InspectorDock::get_singleton()->get_inspector()->edit(nullptr);
```

Comment:
This is what `_push_item(nullpr)` does, no? 🤔
Although it only works with a single node for some reason.

Reply:
Ok you can delete this line, see #74795

Reply:
@SaracenOne This comment still needs addressing (especially now that the linked PR was merged). Then we should be good to go.

Reply:
Bump.